pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -996,7 +996,8 @@ func (l *LibvirtDomainManager) SyncVMI(vmi *v1.VirtualMachineInstance, allowEmul
 	}
 
 	if vmi.IsRunning() {
-		if err := hotplugVirtioInterface(vmi, c, dom, domain); err != nil {
+		vim := newVirtIOInterfaceManager(c, dom, netsetup.NewVMNetworkConfigurator(vmi, cache.CacheCreator{}))
```

Comment:
nit: Please consider renaming `vim` to a longer name like `networkInterfaceManager`.
`vim` is very close to `vmi` (which is very common in this file) and IMO this similarity could be a bit confusing.

Reply:
Done.